### PR TITLE
[WIP] Fix UI build in all-in-one docker container

### PR DESCRIPTION
**Pull request by black-adder**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ env:
     - secure: mX3/Gwv5esCjU9GfqDaXbTslA0UOY9q45+rnW58X6z1tY2pFOts5uhtLve3YneGn3IN0n+m5mcljim72vKNQaPJCSodYfcfrqth2YqkYTXNWykOIdlLs1u58pYYGRXJaAlD4EgQysB/nAaQYO00/n2tBq4vEmu0dikbJer97smStkokLBiz42tJbvY6mikqnHHbRAA4dw5MHwYikDkI82oKxUsB8horetMmXgwTkT3AiE5UufXm3iYxpZ48KTYBW8WbQSQV1T1nriisZVErbYh69QLaLrEK3/WYiN3r4fOag4JvMA5GuPpuVGQfKzXUuFptG54VBbPKCgmcXA/DlaEQ6HILsfHdl29/AXHa59dVjWGfQ5f/N4VWaIVXhE0qWo+/WIoyr7SaLybyGVe3gf1vDaDFlfVlrE7j13H+XKiwzfOeaqnTrcNc/MSCFCtnUbjvX4UlXCbE4gVbCuEsjAPQ8mI/MMaCWLxhaYTW/cSyGRTGMLwe4Q8uAPd3BQat63byLo3BfK+icyw24RbSGBDRu5FZORK164MMjr0+UKVT3gh5/JdFxLkdOGjJL/9QuppNShI5GBIctIg15AQJNkhSLm8DxUTNHhjjBgXzqNhDVmUQLQVgqCWlHh1HVF5eiIQXBNGBE1Lw9v9dGTUHlPUUqeJKsxEyYWfZuY4tNhzU=
 
 install:
-  - nvm use 4
+  - nvm use 6
```
Comment (yurishkuro) [inline .travis.yml]: I am surprised we need this. The `.nvmrc file in jaeger-ui says `4`, and that's what we've been using internally.

Comment (black-adder) [inline .travis.yml]: yeah I don't think this is the issue, just playing around with configs. I'll ping you when the PR is ready.

Comment (black-adder) [inline .travis.yml]: I think the issue is related to this: http://stackoverflow.com/questions/43334968/uncaught-error-react-proptypes-type-checking-code-is-stripped-in-production

I'll add a new branch to the UI that clamps the react version and try to build the docker image with that.

**Comment by black-adder** [comment]
Clamping to 15.4.2 didn't work. Gonna have to wait until the react devs resolve the issue with 15.5.0

**Pull request closed without merging**